doc/source/getting_started.rst

```diff
@@ -0,0 +1,190 @@
+Getting started
+===============
+
+Overview
+--------
+
+This package facilitates working with data templates that follow the
+format developed by the `Integrated Assessment Modeling Consortium
+(IAMC) <https://www.iamconsortium.org>`__. It supports validation of
+scenario data and region processing, which consists of renaming and
+aggregation of model “native regions” to “common regions” used in a
+project.
+
+A **DataStructureDefinition** class contains **CodeLists** for
+*variables* (including units) and *regions* to be used in a model
+comparison or scenario exercise following the IAMC data format.
+
+A **CodeList** is a list of “allowed terms” (or codes), where each term
+can have several attributes (e.g., description, unit, parent region).
+
+A **RegionAggregationMapping** is a mapping that defines on a per-model
+basis how model native regions should be renamed and aggregated to
+comparison regions.
+
+A **RegionProcessor** is a class that holds a list of
```

Comment:
If the class "DataStructureDefinition" and "RegionAggregationMapping" gets explained in more detail, doesn't it make also sense to add the "RegionProcessor" class? If so I would drop "CodeLists" and move it under the "DataStructureDefinition" class. Then it maybe make sense to only use the classes as headings and skip the "Overview" heading and "The pyam package". Or maybe I just don't understand the structure of this page :)

Reply:
I'm not sure I fully understand what you mean.
Do you mean adding a dedicated section for the RegionProcessor class? That would probably be a good idea.
I would vote for keeping the overview as it serves both as on overview and a glossary to explain the terms used in the following to the user.

Reply:
Yes, I meant having for example the following structure analogue to the API documentation:

## Getting started
This package facilitates working with data templates that follow the format developed by the Integrated Assessment Modeling Consortium (IAMC). It supports validation of scenario data and region processing, which consists of renaming and aggregation of model “native regions” to “common regions” used in a project.

### DataStructureDefinition 

A DataStructureDefinition class contains CodeLists for variables (including units) and regions to be used in a model comparison or scenario exercise following the IAMC data format.
A CodeList is a list of “allowed terms” (or codes), where each term can have several attributes (e.g., description, unit, parent region).
...

### RegionAggregationMapping 
...

### RegionProcessor 
...

But as I said maybe I don't understand why the page is structured as it is :)

Reply:
This page is getting quite long for a "getting started" page... I'm afraid that this might scare off potential users.

Reply:
Shortened the getting started page to a minimum working example.